at-quannguyen/W3/dictionary.rb

```diff
@@ -0,0 +1,111 @@
+require 'time'
+# class dictionary
+class Dictionary
+  def initialize
+    @list_dictionary = {}
+    @id = 1
+  end
+
+  def add(dict, des)
+    if Dictionary.check_empty_by_vocab(@list_dictionary, dict)
+      @list_dictionary[@id] =
+        { dict: dict, des: des, lookup: 0, history_lookup: 0 }
+      @id += 1
+      return true
+    end
+    false
+  end
+
+  def lookup(vocab)
+    result = Dictionary.find_dict_by_vocab(@list_dictionary, vocab)
+    if result
+      sleep 0.5
+      result[1][:lookup] += 1
+      result[1][:history_lookup] = Time.new.strftime('%m/%d/%Y %H:%M:%S')
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/6f22e2e17b7a320ef0dcf3e430a8fa8270c1e786/at-quannguyen/W3/dictionary.rb#L24"></span>Rails/TimeZone: Do not use <code>Time.new</code> without zone. Use one of <code>Time.zone.now</code>, <code>Time.current</code>, <code>Time.new.in_time_zone</code>, <code>Time.new.utc</code>, <code>Time.new.getlocal</code>, <code>Time.new.xmlschema</code>, <code>Time.new.iso8601</code>, <code>Time.new.jisx0301</code>, <code>Time.new.rfc3339</code>, <code>Time.new.httpdate</code>, <code>Time.new.to_i</code>, <code>Time.new.to_f</code> instead. (https://github.com/rubocop-hq/rails-style-guide#time, http://danilenko.org/2012/7/6/rails_timezones)</td>
    </tr>
  </tbody>
</table>